src/main/java/com/nguyenquyhy/discordbridge/DiscordBridge.java

```diff
@@ -32,7 +32,7 @@
  * Created by Hy on 1/4/2016.
  */
 @Plugin(id = "discordbridge", name = "Discord Bridge", version = "2.3.0",
-        description = "A Sponge plugin to connect your Minecraft server with Discord", authors = {"Hy", "Mohron"})
+        description = "A Sponge plugin to connect your Minecraft server with Discord", authors = {"Hy", "Mohron, Axle2005"})
```

Comment:
Honestly :P If he wants to deny this one commit, I don't care. The rest of the PR is good, but I'm still going to be hobbling along on this plugin anyway.